Makefile

```diff
@@ -146,7 +146,7 @@ format-go:
 	gofmt -s -w go/
 
 lint-go: compile-protos-go
-	go vet ./go/internal/feast ./go/cmd/goserver
+	go vet ./go/internal/feast ./go/cmd/server
```

Comment:
Are gonna add other linters or formatters here?

Reply:
will add in future PR

should we be having a discussion on what linter to use? doesn't look to me that there's a canonical option, I've been using staticcheck